docs/manual_curation.md

```diff
@@ -0,0 +1,126 @@
+# Manual trait curation
```

Comment:
See notes about Markdown style in `docs/build.md`.

Reply:
Fixed as well